vars/withPipeline.groovy

```diff
@@ -107,65 +106,42 @@ def call(type, String product, String component, Closure body) {
         }
 
         onPR {
-          onTerraformChangeInPR{
-            sectionDeployToEnvironment(
-              appPipelineConfig: pipelineConfig,
-              pipelineCallbacksRunner: callbacksRunner,
-              pipelineType: pipelineType,
-              subscription: subscription.nonProdName,
-              aksSubscription: aksSubscriptions.aat,
-              environment: environment.nonProdName,
-              product: product,
-              component: component,
-              tfPlanOnly: true
-            )
-
+          onTerraformChangeInPR {
             def githubApi = new GithubAPI(this)
-
+            def prCache = githubApi.refreshPRCache() // e.g. demo, perftest, ithc, etc
+            def branchName = branch.branchName.toLowerCase() // could be PR-123 or #58
```

Comment:
we use PR-123 instead of branchName because thats how it recognises the format of a PR. I don't think this is necessary.